resources/styles/global/loadable.less

```diff
@@ -0,0 +1,29 @@
+@-webkit-keyframes fadeIn { from { opacity: 0; } to { opacity: 1; } }
+@-moz-keyframes fadeIn { from { opacity: 0; } to { opacity: 1; } }
+@keyframes fadeIn { from { opacity: 0; } to { opacity: 1; } }
+
+.refresh-button-delayed {
```

Comment:
`<Loadable>` generates elements with classes (`-loading` and `-error`). Would it make sense to change these classes to `.loadable.loading`? Then, is there a need to explicitly have a `refresh-button-delayed` class?